plio/tests.py

```diff
@@ -439,6 +439,56 @@ def test_org_user_can_list_other_user_items(self):
         # set db connection back to public (default) schema
         connection.set_schema_to_public()
 
+    def test_guest_cannot_update_items(self):
+        # unset the credentials
+        self.client.credentials()
+        # update item
+        response = self.client.put(
+            f"/api/v1/items/{self.item.id}/", {"time": 2, "plio": self.plio.id}
+        )
+        self.assertEqual(response.status_code, status.HTTP_401_UNAUTHORIZED)
+
+    def test_user_can_update_own_items(self):
+        # update item
+        response = self.client.put(
+            f"/api/v1/items/{self.item.id}/", {"time": 2, "plio": self.plio.id}
+        )
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+
+    def test_user_cannot_update_other_items(self):
+        # create plio from a user 2
+        new_plio = Plio.objects.create(
+            name="Plio 2", video=self.video, created_by=self.user_2
+        )
+
+        # create item from a user 2
+        new_item = Item.objects.create(type="question", plio=new_plio, time=1)
+
+        # update item
+        response = self.client.put(
+            f"/api/v1/items/{new_item.id}/", {"time": 2, "plio": self.plio.id}
+        )
+        self.assertEqual(response.status_code, status.HTTP_403_FORBIDDEN)
```

Comment:
as the comment above, can also check that the item was NOT updated. But not super necessary here as 403 is already being returned